src/index.html

```diff
@@ -1,13 +1,435 @@
+              <img
+                class="logo__image"
+                src="images/logo.svg"
+                alt="MET"
+              >
+            </a>
+            <div class="header__wrapper">
+              <a href="/" class=" icon icon--cross header__wrapper__menu-opener"></a>
+            </div>
+          </div>
+
+          <ul class="menu__list">
+            <li class="menu__item">
+              <a href="#" class="menu__link">Home</a>
+            </li>
+            <li class="menu__item">
+              <a href="#hours" class="menu__link">Museum hours</a>
+            </li>
+            <li class="menu__item">
+              <a href="#now-on-view" class="menu__link">Now on view</a>
+            </li>
+            <li class="menu__item">
+              <a href="#explore" class="menu__link">More to Explore Online</a>
+            </li>
+            <li class="menu__item">
+              <a href="#contacts" class="menu__link">Contact us</a>
+            </li>
+          </ul>
+          <p class="menu__label">The Met Fifth Avenue</p>
+          <a href="tel:+1 212-535-7710" class="menu__call menu__call--phone">+1 212-535-7710</a>
+          <p class="menu__label">The Met Cloisters</p>
+          <a href="tel:+1 212-923-3700" class="menu__call menu__call--phone--two">+1 212-923-3700</a>
+          <a href="tel:+1 212-535-7710" class="menu__call menu__call--action">call to order</a>
+        </div>
+      </div>
+    </nav>
+    <main>
+      <section class="page__section hours" id="hours">
+        <div class="container">
+          <div class="grid grid--tablet">
+            <div class="grid__item grid__item--tablet-1-3 grid__item--desktop-1-5">
+              <h7 class = "hours__link ">
+                <a href="#" class="link">Plan your visit</a>
+              </h7>
+              <h2 class = "hours__title">Museum hours</h2>
+            </div>
+
+            <div class="grid__item grid__item--tablet-4-6 grid__item--desktop-6-12">
+              <div class="hours__content">
+                <div class="hours__content-subtitle"> The Met Fifth Avenue</div>
+                <div class="content hours__content-hour">10 am - 5 pm<br>10 am - 9 pm<br>Closed</div>
+                <div class="content hours__content-day">Sun, Mon, Tue, Thu<br>Fri, Sat<br>Wen</div>
+                <div class="content hours__content-closed">Closed on<br>Thanksgiving Day, Dec 25, Jan 1, and the first Monday in May.</div>
+              </div>
+            </div>
+            <div class="grid__item grid__item--tablet-4-6 grid__item--desktop-6-12">
+              <div class="hours__content">
+                <div class="hours__content-subtitle"> The Met Cloisters</div>
+                <div class="content hours__content-hour">10 am – 4.30 pm<br>Closed</div>
+                <div class="content hours__content-day">Thu - Tue<br>Wen</div>
+                <div class="content hours__content-closed">Closed on<br>Thanksgiving Day, Dec 25, Jan 1.</div>
```

Comment:
"Closed on" text should be on bottom
![image](https://user-images.githubusercontent.com/37161315/170513122-2a533d53-35c0-4f0b-9164-47a459214228.png)